### ISSUE TITLE
Auto-publishing time UTC vs local

### ISSUE THREAD

**Issue by peterdesmet** [gbif/ipt#2464]
For one of our resources we have an auto-publication set to Wednesday at 12:00:

<img width="477" alt="Screenshot 2024-07-12 at 12 09 05" src="https://github.com/user-attachments/assets/14f581b9-1933-4ed6-b705-9f7d4f501b36">

According to the log file, the resource gets published at `10:00`, but I don't know if that is UTC or local time:

```
10:00:03 Archive generation started for version #1.119
10:00:03 Start writing data file for Darwin Core Occurrence
```

As a user I expect the resource to be published at 12:00 local time, but I have the impression the `12:00 UTC +2` is converted to `10:00 UTC` and the resource gets published at 10:00 local time.

@mike-podolskiy90

1. Can you investigate if the resource is indeed auto-published at the defined local time?
2. Could you align how times are shown in the log file with how they are shown in the auto-publishing? I think it's fine everything is UTC, but then it would be good to indicate that and maybe show UTC in addition to the local time in the auto-publishing settings.

**Comment by dbloom** [comment]
I appreciate @peterdesmet for posting this. Over the years, I also have noted the discrepancy between when I set (and expect) publication to occur in my local time zone and the timestamp that is indicated by GBIF when publication actually occurs. The indication is that publication actually occurs in the time zone of the server and not my local time. It was never important enough for me to report, but I am reminded of it every time I set a publication date and time or need to confirm that a resources was published on schedule.

Also, now that the VN IPT is hosted by GBIF, if publication time is intended to be local time, it might be necessary to add an option to select a time zone or UTM. As I think about now, I'm guessing the cloud IPTs all publish on CEST times.
